### PR TITLE
[ARTS-382] Add trigger for deployment wf on main-nightly

### PR DESCRIPTION
**Pull request by mdp0**
## Description
Trigger the mts-deployment-config repo's workflow, passing the mainnightly trial config as a parameter

### Changes
- [ARTS-382] - auto deploy main-nightly build to 'dev' env

### Checklist:

- [x] Branch name follows convention (feature/arts-#-short-name OR fix/arts-#-short-name)
- [x] I have included a short-meaningful title, description and changes for this PR

[ARTS-382]: https://ndph-arts.atlassian.net/browse/ARTS-382

### CODE REVIEW

File: .github/workflows/docker-build-push.yml [hunk 1]
```diff
@@ -120,6 +120,20 @@ jobs:
           docker tag ${{ steps.check_docker_images.outputs.runtime_tag }} $TO_TAG
           docker push $TO_TAG
 
+      - name: Deploy main-nightly image
+        if: github.ref == 'refs/heads/main' && github.event_name == 'schedule'
+        uses: convictional/trigger-workflow-and-wait@v1.3.0
+        with:
+          owner: ndph-arts
+          repo: mts-deployment-config
+          github_token: ${{ secrets.DEPLOYMENT_TRIGGER_PAT }}
```
Comment (eladiw) [inline .github/workflows/docker-build-push.yml:129]: DEPLOYMENT_TRIGGER_PAT**H**?

Comment (eladiw) [reply .github/workflows/docker-build-push.yml:129]: notice the missing H

File: .github/workflows/docker-build-push.yml [hunk 2]
```diff
@@ -120,6 +120,20 @@ jobs:
           docker tag ${{ steps.check_docker_images.outputs.runtime_tag }} $TO_TAG
           docker push $TO_TAG
 
+      - name: Deploy main-nightly image
```
Comment (eladiw) [inline .github/workflows/docker-build-push.yml:123]: Shouldn't this nightly version have its own workflow yml file? a scheduled one?

**Review by eladiw — CHANGES_REQUESTED** [review]
few comments

File: .github/workflows/docker-build-push.yml (hunk 1 above)
Comment (mdp0) [reply .github/workflows/docker-build-push.yml:129]: PAT = personal access token

File: .github/workflows/docker-build-push.yml (hunk 2 above)
Comment (mdp0) [reply .github/workflows/docker-build-push.yml:123]: We could put it in a separate file, I figured that we would want to push it on the same schedule as the build. Easy either way.

**Review by darrengsmithson — APPROVED** [review]

**Review by eladiw — APPROVED** [review]

**Pull request merged**